bigbluebutton/greenlight
# Share Rooms Site Setting

## Description

- Toggle for Share Rooms site setting now works
- When it is set to off, the Shared Access tab is not shown on to Room page
## Testing Steps

- Create room 
- Go to Admin Panel
- Go to Site Settings
- Toggle on/off
- Check to see that share access tab on room page appears/disappears based on value
## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/38328371/179601657-ba5f5ee8-0fa8-40dc-bfde-11aacbd1aa7a.png)

## TODO
- Some front end touch-ups will be made once more settings are added to page
